tests/test_ops_nonlinear.py

```diff
@@ -67,7 +67,7 @@ def leaky_relu(x, alpha=0.01):
     assert_onnx_import_equals_callable('LeakyRelu', leaky_relu, [-2, -1., 0., 1., 2.], alpha=0.5)
     assert_onnx_import_equals_callable('LeakyRelu', leaky_relu, [0.])
     assert_onnx_import_equals_callable('LeakyRelu', leaky_relu,
-                                       [-0.9, -0.8, -0.7, -0.4, -0.3, -0.2, -0.1], alpha=1)
+                                       [-0.9, -0.8, -0.7, -0.4, -0.3, -0.2, -0.1], alpha=1.)
```

Comment:
Why change from int to float?

Reply:
Onnx model.check fails. According to Op documentation this value should be floating point.